src/item.cpp

```diff
@@ -6188,7 +6188,9 @@ static int phys_resist( const item &it, damage_type dt, clothing_mod_type cmt,
             base_resistance = iter->second;
         }
 
-        float damaged_resistance = base_resistance * eff_thickness / it.get_thickness();
+        // We can have 0 thickness items, so need to check for it to ensure we don't get NaN in calcs
+        int thickness = it.get_thickness();
+        float damaged_resistance = ( 0 != thickness ) ? base_resistance * eff_thickness / thickness : 0.0f;
```

Comment:
```suggestion
        const int thickness = it.get_thickness();
        const float damaged_resistance = ( thickness == 0 ) ? 0.0f : base_resistance * eff_thickness / thickness;
```

1. adding `const` improves readability by removing moving/mutable parts in code.
2. (optional) it might be a good idea to use `variable != 0` over `0 != variable` as variables are usually more important than numbers.
3. (optional) it also might be a good idea to prefer `==` over `!=` as it may reduces cognitive load by one (is _not_ something -> is something)